app/models/card.rb

```diff
@@ -15,39 +19,24 @@ class Card < ActiveRecord::Base
 
   mount_uploader :image, CardImageUploader
 
-  scope :pending, -> { where('review_date <= ?', Time.now).order('RANDOM()') }
-  scope :repeating, -> { where('quality < ?', 4).order('RANDOM()') }
-
-  def check_translation(user_translation)
-    distance = Levenshtein.distance(full_downcase(translated_text),
-                                    full_downcase(user_translation))
-
-    sm_hash = SuperMemo.algorithm(interval, repeat, efactor, attempt, distance, 1)
-
-    if distance <= 1
-      sm_hash.merge!({ review_date: Time.now + interval.to_i.days, attempt: 1 })
-      update(sm_hash)
-      { state: true, distance: distance }
-    else
-      sm_hash.merge!({ attempt: [attempt + 1, 5].min })
-      update(sm_hash)
-      { state: false, distance: distance }
-    end
-  end
+  scope :pending, -> {
+    where("review_date <= ?",
+    Time.zone.now).order('RANDOM()')
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.